doc/Development_Documentation/05_Objects/03_Working_with_PHP_API.md

```diff
@@ -100,7 +100,7 @@ $entries->setCondition("name LIKE " . $entries->quote("%bernie%")); // make sure
 // some data types support direct filtering
 $entries->filterByName('Jan'); // filters for name='Jan'
 $entries->filterByAge(18, '>='); // filters for age >= 18
-$entries->filterByCity(['New York', 'Chicago'], 'IN (?)'); // filters for city IN ('New York','Chicago')
+$entries->filterByCity([['New York', 'Chicago']], 'IN (?)'); // filters for city IN ('New York','Chicago')
```

Comment:
No need to update example...if data wrapped in array!

Reply:
Ah sorry, I did not want to commit the parameter wrapping in an array to allow the comparison operator to have multiple `?` placeholders.
Now the doc matches the code.